### PR TITLE
Contrib readme

### PR DESCRIPTION
**Pull request by Xorima**
Adding contibutors/sponsers 

[Describe what this change achieves]

## Issues Resolved

[List any existing issues this PR resolves]

## Check List

- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by Xorima** [comment]
@damacus what do you think about it in this style? it should even pass mdlint :D 
Dropped off the contributors image and changed to just a link (it's too large otherwise)

File: README.md [hunk 1]
```diff
@@ -86,6 +88,29 @@ Example wrapper cookbooks:
...
 - [module](documentation/resource_apache2_module.md)
 
+## Contributors
+
+This project exists thanks to all the people who [contribute.](https://opencollective.com/sous-chefs/contributors.svg?width=890&button=false)
+
+### Backers
+
+Thank you to all our backers! 🙏 [Become a backer](https://opencollective.com/sous-chefs#backer)
+![https://opencollective.com/sous-chefs#backers](https://opencollective.com/sous-chefs/backers.svg?width=890)
```
Comment (damacus) [inline README.md]: This could do with being on a second line here. 

I think *think* the layout looks better

**Comment by damacus** [comment]
HAProxy has a good example of a wide image (that you can resize) that doesn't require maintaining when we get new backers :) 

https://github.com/sous-chefs/haproxy

**Comment by Xorima** [comment]
> HAProxy has a good example of a wide image (that you can resize) that doesn't require maintaining when we get new backers :)
> 
> https://github.com/sous-chefs/haproxy

I agree, however there does not appear to be a tier widget available for the corporate sponsor tier
This way for sponsors also lets it link back to them, I understand it's super manual but I don't think we will have 10 sponsors any time soon, so maintenance shouldn't be too much of an issue?

If you are about today on slack lets have a chat and get this ironed out? :)

**Pull request merged**